data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+		"attitude toward"
+			"Republic that won't enter wormhole" 1.0
+	government "Republic that won't enter wormhole"
+		"attitude toward"
+			"Republic" 1.0
+
+	government "Syndicate"
+		"attitude toward"
+			"Syndicate that won't enter wormhole" 1.0
+	government "Syndicate that won't enter wormhole"
+		"attitude toward"
+			"Syndicate" 1.0
+
+	government "Free Worlds"
+		"attitude toward"
+			"Free Worlds that won't enter wormhole" 1.0
+	government "Free Worlds that won't enter wormhole"
+		"attitude toward"
+			"Free Worlds" 1.0
+
+	government "Deep Security"
+		"attitude toward"
+			"Deep Security that won't enter wormhole" 1.0
+	government "Deep Security that won't enter wormhole"
+		"attitude toward"
+			"Deep Security" 1.0
+
+event "hai ladybug available"
+	shipyard "Hai Intermediate"
+		"Ladybug"
+	shipyard "Hai Advanced"
+		"Ladybug"
+
+mission "Hai Reveal: Reputation Reset"
+	landing
+	invisible
+	to offer
+		has "event: pug media coverage"
+	on offer
+		# Sync reputations for aliases of governments:
+		"reputation: Deep Security that won't enter wormhole" = "reputation: Deep Security"
+		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
+		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
+		"reputation: Syndicate that won't enter wormhole" = "reputation: Syndicate"
+
+		# Make sure the new governments start with appropriate rep:
+		"reputation: Human Alliance that won't enter wormhole" = 1000
+		"reputation: Human Alliance" = 1000
+		"reputation: Hai (Friendly Unfettered)" = 1000
+		"reputation: Bounty Hunter that Attacks Hai" = -1000
+		"reputation: Philosophers" = 1000
+		"reputation: Pirate (Devil-Run Gang)" = -1000
+		fail
+
+mission "Event: Pug Media Coverage"
+	landing
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "spaceport"
+	to offer
+		has "event: pug media coverage"
+	on offer
+		event "hai secret leaks" 10 15
+		conversation
+			`As you walk through the spaceport, you can't help but notice that coverage on the events of the Pug invasion still continues to dominate headlines, even though it's been weeks now.`
+			`	Looking around, you see that experts and officials have given way to talking heads and theorists. Some of them are saying what you yourself suspect: that the Pug invasion was a ploy to cause humanity to band together, learn to resolve differences without conflict, and ensure the sorts of extremism that lead to the events precipitating the war would not be tolerated.`
+			`	Others...`
+			`	You sigh deeply as you watch some paranoid fringe world politician demand that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known, and that it is the responsibility of the government to find out and inform the people.`
+			`	Your thoughts, you keep to yourself.`
+				decline
+
+event "hai secret leaks"
+
+
+conversation "hai reveal free worlds news"
```

Comment:
I recall you mentioning in the past that you liked there to be examples of different functionality presented occasionally. I think this is a particularly good example of named conversations being shared.

But also on reflection I recall another reason for doing it this way:
When other campaigns get added there will be even more possible combinations. As such the objective here is to matrix it out, so that all geographic starting points are defined by the missions - and then the conversations handle the combination of branches required for each campaign in a given geographic area.

This makes it clearer and easier to work on and expand in the future when required.

As such I would like to maintain this structure.